README.rst

```diff
@@ -166,6 +166,43 @@ dictionaries you would do the following:
     connect("username", "password", representation=dict_representation)
 
 
+Reports
+-------
+
+Retrieving report data has 3 steps:
+
+1. Creating a report
+2. Check if the report is ready to download
+3. Download the report
+
+The ``download`` method on ``Report`` object takes care of checking if the report is
+available for download and retires it by default for 3 times with an interval of 1 second.
+The number of retries can be overridden by passing the parameter ``retry_count`` to the ``download``
```

Comment:
Maybe you could put that sentence after the code block. Then you could take your commented example and do another code block as an example of that sentence.

Reply:
So basically you would just put line 180 between lines 2010 and 202, add another `code-block` directive and uncomment the code.